tests/cloudpickle_test.py

```diff
@@ -42,7 +43,8 @@
     tornado = None
 
 import cloudpickle
-from cloudpickle.cloudpickle import _find_module, _make_empty_cell, cell_set
+from cloudpickle.cloudpickle import _find_module, _make_empty_cell, cell_set,\
+        _dynamic_modules_globals
```

Comment:
Please use a new `from .... import line` instead of multiline imports: I find it more readable and it makes it easier to avoid merge conflict.